lib/post_request_task/tests.py

```diff
@@ -0,0 +1,88 @@
+from django.core.signals import request_finished, request_started
+from django.test import TestCase
+
+from mock import Mock, patch
+from nose.tools import eq_
+
+from lib.post_request_task.task import (task, _get_task_queue, _discard_tasks,
+                                        _stop_queuing_tasks)
+
+
+task_mock = Mock()
+
+
+@task
+def test_task():
+    task_mock()
+
+
+class TestTask(TestCase):
+
+    def setUp(self):
+        super(TestTask, self).setUp()
+        task_mock.reset_mock()
+        _discard_tasks()
+
+    def tearDown(self):
+        task_mock.reset_mock()
+        _discard_tasks()
+        _stop_queuing_tasks()
+        super(TestTask, self).tearDown()
+
+    def _verify_task_filled(self):
+        queue = _get_task_queue()
+        size = len(queue)
+        eq_(size, 1, 'Expected 1 task in the queue, found %d' % size)
+        cls, args, kwargs = queue[0]
+
+    def _verify_task_empty(self):
+        assert len(_get_task_queue()) == 0, (
+            'Expected empty queue, got %s: %s' % (len(_get_task_queue()),
+                                                  _get_task_queue()))
+
+    def test_task(self):
+        test_task.delay()
```

Comment:
I'd like to see an added test that makes sure args/kwargs sent to `delay(...)` are passed to the original task